### PR TITLE
VIH-8967 - Add Notification API Config

### PR DESCRIPTION
**Pull request by v1ferrarij**
### JIRA link (if applicable) ###

VIH-8967

### Change description ###

Add Notification API Config
- Secrets

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: terraform/09-video-hearing-core/20-main.tf [hunk 1]
```diff
@@ -99,7 +99,7 @@ module "KeyVault_Secrets" {
       content_type = "secret"
     },
     {
-      name         = "connectionstrings--vhnotification"
+      name         = "connectionstrings--vhnotificationsapi"
```
Comment (chris-pateman) [inline terraform/09-video-hearing-core/20-main.tf:102]: did you not have me change this?

File: terraform/09-video-hearing-core/20-main.tf [hunk 2]
```diff
@@ -134,6 +134,12 @@ module "KeyVault_Secrets" {
       tags         = local.common_tags
       content_type = "secret"
     },
+    {
+      name         = "notifyconfiguration--callbacksecret"
```
Comment (chris-pateman) [inline terraform/09-video-hearing-core/20-main.tf:138]: for the PR pipeline you can get the secret from here https://dev.azure.com/hmcts/Video%20Hearings/_library?itemType=VariableGroups&view=VariableGroupView&variableGroupId=316&path=govuk-notify-dev

for the flux release, this is dependent on [VIH-9086](https://tools.hmcts.net/jira/browse/VIH-9086)
for now I would manually add a secret as a temp

Comment (v1ferrarij) [reply terraform/09-video-hearing-core/20-main.tf:138]: Yeah but them values look the same as what is in the current implementation.. after discussion with Murali earlier, new values need to be created for SDS.

would rather leave this in terraform and just update/recreate the secret when we have the correct values.

**Review by chris-pateman — APPROVED** [review]
LGTM

**Pull request merged**